src/IxMilia.Dxf/DxfFile.cs

```diff
@@ -439,39 +439,48 @@ private void EnsureStyleObjects()
 
         private void EnsureTableItems()
         {
+#if NET35
+            var existingDimStyles = GetExistingNames(DimensionStyles.Cast<DxfSymbolTableFlags>());
```

Comment:
The calls to `.Cast<T>()` are simple enough I'm OK if these are simply inlined everywhere; I see no need to separate out something so simple.